src/pkg/bundle/dev.go

```diff
@@ -47,8 +49,13 @@ func (b *Bundle) CreateZarfPkgs() {
 				}
 			}
 			// create local zarf package if it doesn't exist
-			if !packageFound {
-				os.Args = []string{"zarf", "package", "create", pkgDir, "--confirm", "-o", pkgDir, "--skip-sbom"}
+			if !packageFound || b.cfg.DevDeployOpts.ForceCreate {
+				if len(b.cfg.DevDeployOpts.Flavor) != 0 {
+					b.setPackageFlavor(&pkg)
```

Comment:
`b.setPackageFlavor` mutates `pkg`, but `b.setPackageRef` returns the `pkg` and reassigns. Personally I prefer the immutable way that `b.setPackageRef` does it, either way thought we should make it consistent unless there's a reason not to